### PR TITLE
test: Don't run BackupSuite concurrently

### PR DESCRIPTION
**Pull request by lmars**
It is proving disruptive to other tests.

### CODE REVIEW

**Comment by titanous** [comment]
LGTM

**Comment by lmars** [comment]
@titanous I had to add a few more fixes due to the fact I moved the default cluster to listen on just `eth0` (rather than `0.0.0.0` which prevented the backup clusters from booting).

File: host/volume/api/http.go [hunk 1]
```diff
@@ -17,15 +19,21 @@ import (
 const snapshotContentType = "application/vnd.zfs.snapshot-stream"
 
 type HTTPAPI struct {
-	cluster *cluster.Client
-	vman    *volumemanager.Manager
+	vman *volumemanager.Manager
+
+	cluster    *cluster.Client
+	clusterMtx sync.Mutex
```
Comment (titanous) [inline host/volume/api/http.go]: Can this be a `sync.Atomic` instead?

**Comment by titanous** [comment]
LGTM

**Pull request merged**